### ISSUE TITLE
Ordering process should take less than 10 min

### ISSUE THREAD

**Issue by patrikgrenfeldt** [Clinical-Genomics/cg#1310]
## Description

Sometimes with huge orders the ordering takes longer than 10min and then the full order is not saved in status-db.

## Suggested solutions

- Have the different steps (ticketing, lims, status-db) as transaction so we can roll-back all of them in case of failure
- Call the ticket service asynchronously so we don't spend time waiting for the service to complete (>minutes sometimes)  
- rearrange the different step so that our customers don't think the order went through when it didn't

## Suggested solution

- Call the ticket service asynchronously - seems like a thing that should be possible to implement

## This can be closed when

- When an order can be placed (including ticketing) for 200 cases with one sample each

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by Mropat** [comment]
This is to be addressed in new orderportal.
If there is a bug currently affecting orderportal, new issue(s) should be opened with concrete descriptions of the bug
